osu.Game/Overlays/SkinEditor/SkinSelectionHandler.cs

```diff
@@ -102,27 +137,19 @@ protected override void DeleteItems(IEnumerable<ISerialisableDrawable> items) =>
 
         protected override IEnumerable<MenuItem> GetContextMenuItemsForSelection(IEnumerable<SelectionBlueprint<ISerialisableDrawable>> selection)
         {
-            var closestItem = new TernaryStateRadioMenuItem(SkinEditorStrings.Closest, MenuItemType.Standard, _ => applyClosestAnchors())
-            {
-                State = { Value = GetStateFromSelection(selection, c => !c.Item.UsesFixedAnchor) }
-            };
+            closestAnchor = new TernaryStateRadioMenuItem(SkinEditorStrings.Closest, MenuItemType.Standard, _ => applyClosestAnchors());
+            fixedAnchors = createAnchorItems(applyFixedAnchors).ToArray();
 
             yield return new OsuMenuItem(SkinEditorStrings.Anchor)
             {
-                Items = createAnchorItems((d, a) => d.UsesFixedAnchor && ((Drawable)d).Anchor == a, applyFixedAnchors)
-                        .Prepend(closestItem)
-                        .ToArray()
+                Items = fixedAnchors.Prepend(closestAnchor).ToArray()
             };
 
             yield return originMenu = new OsuMenuItem(SkinEditorStrings.Origin);
 
-            closestItem.State.BindValueChanged(s =>
-            {
-                // For UX simplicity, origin should only be user-editable when "closest" anchor mode is disabled.
-                originMenu.Items = s.NewValue == TernaryState.True
-                    ? Array.Empty<MenuItem>()
-                    : createAnchorItems((d, o) => ((Drawable)d).Origin == o, applyOrigins).ToArray();
```

Comment:
Note that this is replaced by disabling all 9 items instead. This is because this code is janky at best on master right now (if you right-click closest anchor, the origin menu won't disable until you move away from the anchor submenu somewhere else).